docs/satellite-clone.md

```diff
@@ -1,19 +1,22 @@
-## Cloning a Satellite server
+## Cloning or Migrating a Satellite server
```

Comment:
Similar to the other comment, This phrasing makes Migrating separate from Cloning.  I would also vote to keep this as `Cloning a Satellite Server`